test/process.cpp

```diff
@@ -330,8 +337,8 @@ TEST_CASE("Launch 2 process at the time with launcher in separeted thread")
     {
```

Comment:
Process finish before timeout. It is not what we want to test (same as previous test)